src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -1,10 +1,12 @@
 // (C) 2019-2020 GoodData Corporation
 import * as React from "react";
 import { render } from "react-dom";
-
-import cloneDeep = require("lodash/cloneDeep");
 import get = require("lodash/get");
 import set = require("lodash/set");
+import tail = require("lodash/tail");
+import isEmpty = require("lodash/isEmpty");
+import includes = require("lodash/includes");
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
ES6 also has `includes`

Reply:
Not supported IE :(

Reply:
sadly, IE doesn’t support it